src/entity/src/user.js

```diff
@@ -273,10 +273,8 @@ export default class User {
     return request.execute()
       .then(response => response.data)
       .then((data) => {
-        if (credentials[socialIdentityAttribute]) {
-          data[socialIdentityAttribute] = credentials[socialIdentityAttribute];
-        }
-
+        data = assign(data, credentials);
+        delete data.password;
```

Comment:
Where does the password get populated from? I'm just asking to make sure it's not persisted anywhere.